.github/workflows/test.yml

```diff
@@ -175,6 +175,44 @@ jobs:
         $env:_LIMA_WINDOWS_EXTRA_PATH = 'C:\Program Files\Git\usr\bin'
         bash.exe -c "./hack/test-templates.sh templates/experimental/wsl2.yaml"
 
+  windows-qemu:
+    name: "Windows tests (QEMU)"
+    runs-on: windows-2022-8-cores
+    timeout-minutes: 30
+    steps:
+    - name: Set gitconfig
+      run: |
+        git config --global core.autocrlf false
+        git config --global core.eol lf
+    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683  # v4.2.2
+      with:
+        # To avoid "can't parse builtin Lima version" errors
+        fetch-depth: 0
+    - uses: actions/setup-go@0aaccfd150d50ccaeb58ebd88d36e91967a5f35b  # v5.4.0
+      with:
+        go-version: 1.24.x
+    - name: Unit tests
+      run: go test -v ./...
+    - name: Make
+      run: make
+    - name: Install winget
+      # TODO remove this step for Windows Server 2025 based runners
```

Comment:
Windows 2025 seems now available:
- https://github.com/actions/runner-images/issues/11742

Reply:
I've just created a new runner type for this, so we can try it out:

![CleanShot 2025-04-30 at 15 12 54@2x](https://github.com/user-attachments/assets/bc4625de-b475-4d9b-916a-dc78c0a18750)

Reply:
Are we sure we actually need 8 cores? Or maybe we just start with a larger runner, and try later to see how much worse it becomes to run on a standard runner?

The 4-core runner is also limited to 16GB. That is plenty on macOS; not sure what is needed on Windows.

Reply:
Why do we need to create a custom runner?

Reply:
So it becomes available in our org? Somebody must have created the `windows-2022-8-cores` runner too, as it is also a custom entry for the `lima-vm` org (I don't think it was me). CNCF has only created custom Linux runners:

![CleanShot 2025-05-01 at 10 48 15@2x](https://github.com/user-attachments/assets/133a4b95-3f4b-4393-8abc-a62ee91a70ce)

Reply:
[About Ubuntu and Windows larger runners](https://docs.github.com/en/actions/using-github-hosted-runners/using-larger-runners/about-larger-runners#about-ubuntu-and-windows-larger-runners):

> Larger runners with Ubuntu or Windows operating systems are configured in your organization or enterprise.

[About macOS larger runners](https://docs.github.com/en/actions/using-github-hosted-runners/using-larger-runners/about-larger-runners#about-macos-larger-runners):

> No additional configuration is required.

Reply:
> Probably it should just work with the standard runner now

Ok, let's try the standard runner first then, to avoid the additional cost.

Reply:
I used standard runners for my development. It is possible to start from them and then compare if more cores are really beneficial.